connection.go

```diff
@@ -549,22 +567,22 @@ func (c *Connection) Insecure() bool {
 
 // AccountsMgmt returns the client for the accounts management service.
 func (c *Connection) AccountsMgmt() *accountsmgmt.Client {
-	return accountsmgmt.NewClient(c, "/api/accounts_mgmt", "/api/accounts_mgmt")
+	return accountsmgmt.NewClient(c.transport, "/api/accounts_mgmt", "/api/accounts_mgmt")
```

Comment:
Not sure I understand this.
`NewClient` expects a `*Connection` but you are sending it `RoundTripper`

Reply:
`NewClient` expects RoundTripper.